cellfinder/napari/detect/detect.py

```diff
@@ -319,6 +320,11 @@ def widget(
             Increase logging
         reset_button :
             Reset parameters to default
+        detection_torch_device : str
```

Comment:
I think this should be a checkbox, for a better user experience. I think it's label should be "Use GPU" and it should be ticked by default if `torch.cuda.is_available()` and greyed out if not.